### PR TITLE
created Settings, bottom navigation and responsiveness

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check

`src/components/bottomNavigation/BottomNavigation.jsx`
`src/pages/Setting/Setting.jsx`
`src/pages/Setting/settingComponent.js`

## [Live](https://share-arts-git-settings-developers-codz.vercel.app/)

Thanks for reviewing my PR :)

### CODE REVIEW

File: src/components/aside/Aside.jsx [hunk 1]
```diff
@@ -12,19 +12,20 @@ export const Aside = () => {
...
-               exact activeStyle={{
-                color: 'red',
-              }}
+            <NavLinks key={id}  to={path}   exact activeStyle={{
+              color: 'red ',
+            }}>
+                <IconWrapper>{icon}</IconWrapper>
+              <NavItem 
+             
                 style={{ color: getAsideColor(theme) }}
```
Comment (Hamza-777) [inline src/components/aside/Aside.jsx:23]: Try avoiding inline css if possible

File: src/components/aside/asideComponent.js [hunk 2]
```diff
@@ -6,25 +6,29 @@ export const NavMenu = styled.ul`
...
-    background-color:#f1e9e7;
-    color:var(--primary-cta) !important;
+  padding: 1rem 1rem 1rem 0.4rem;
 
-  }
+  color: black;
 `;
 
-
+export const IconWrapper = styled.span`
```
Comment (Hamza-777) [inline src/components/aside/asideComponent.js:31]: Styled Components 🤩, Nice choice.

**Review by Hamza-777 — APPROVED** [review]
Amazing work!!

File: src/components/aside/Aside.jsx (hunk 1 above)
Comment (Developers-codz) [reply src/components/aside/Aside.jsx:23]: Okay, Here I am using that so that I can get the color according to theme state

**Review by Dev-AkshatGupta — APPROVED** [review]
All  files been reviewed you have asked for and they are good to merge
Its amazing that you are using styled components.

**Pull request merged**